pkg/injector/sidecar/pod-patch.go

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
nit: filename of some new files are currently in the form of `X-Y.go`, should they rather be `X_Y.go` (or `XY.go`)? https://github.com/golang/go/issues/36060#issuecomment-563435344